sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -149,6 +149,7 @@ class Top(object):
   """Combiners for obtaining extremal elements."""
   # pylint: disable=no-self-argument
 
+  @staticmethod
```

Comment:
I think it's just an issue with Github that this commit shows up here. When merging I (or whoever merges) will make sure that it doesn't go in.